### PR TITLE
Fraud use case

### PR DESCRIPTION
**Pull request by alexxiao007**
This one include the raw data, sample data, python script and output cohort data.

### CODE REVIEW

**Comment by Zrealshadow** [comment]
The main logic of this query:
We consider actions = 'bW9aWZ5X3Bhc3N3b3JK' is the suspected event. 
And we groupby the users when executing suspected event in same clientIP and same date. If the number of users is larger than 1, we regard that these users belong to a cohort.

I don't know whether it is belonged to the concept of cohort. @KimballCai  .
But in the whole there is not birth action and no follow-up events after Birth action. In my opinion, the target of this sql can be achieved simply by SQL query.

Additionally, the dataset is small, only 4 cohorts taken out. It's hard to take it as a test unit for cohort procesing engine

File: datasets/fraud use case/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+User’s account may be taken over by some cyber attackers. To save time and resource, one attacker may steal more than one account within a short time. We can find such stolen accounts by analysing their actions.
+
+### User selection:
+
+Normal accounts seldom change the password. If a group of users who update password using the same IP in the same day, they can be treated as a cohort. It is likely that the accounts may be stolen by the same hackers.
+
+### Birth Criteria:
+
+Users who have update password using the same IP in the same day with others. Group by: IP + action date
```
Comment (KimballCai) [inline datasets/fraud use case/README.md:16]: The analyst should first process the data and add one column to check whether the IP is the same as the others.

File: datasets/fraud use case/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Cohort Matrix:
+
```
Comment (KimballCai) [inline datasets/fraud use case/README.md:19]: I think that we cannot record the ip and the device of the login action. We can only record the times of login. So how can we implement it?

File: datasets/fraud use case/cohort.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+"[25072, 49824]"
```
Comment (KimballCai) [inline datasets/fraud use case/cohort.csv:1]: how can we analyze these results? where is x-axis and y-axis

**Comment by Zrealshadow** [comment]
solved in PR#97

**Pull request closed without merging**